src/bin/triplydb-create-query-job.ts

```diff
@@ -107,29 +89,40 @@ const command = program
     const account = await app.getUser(options.account);
     // check whether account name exists
     await account.getInfo();
-    const queries: { queryId: string; priority?: number }[] = [];
-    for (const query of queryInfo) {
-      const queryAccount = await app.getAccount(query.queryAccountName);
-      const queryId = (await (await queryAccount.getQuery(query.queryName)).getInfo()).id;
-      queries.push({ queryId, priority: query.priority });
-    }
-    const sourceDatasetAccount = await app.getAccount(sourceDatasetAccountName);
-    const sourceDatasetId = (await (await sourceDatasetAccount.getDataset(sourceDatasetName)).getInfo()).id;
-    const targetDatasetAccount = await app.getAccount(targetDatasetAccountName);
-    const targetDatasetId = (await (await targetDatasetAccount.ensureDataset(targetDatasetName)).getInfo()).id;
-    const payload: QueryJobPipelineCreate = {
-      queries: queries,
-      sourceDatasetId: sourceDatasetId,
-      targetDatasetId: targetDatasetId,
-      targetGraphName: options.targetGraphName,
-    };
-    const queryJob: QueryJob = new QueryJob(app, account);
-    try {
-      await queryJob.createQueryJobPipeline(payload, queryInfo);
-    } catch (e) {
-      console.error(e);
-      process.exit(1);
-    }
+    readJson(configFile)
```

Comment:
why are we using `then` instead of simply awaiting the promise? (like you do the line)